src/client/components/input-field/index.css

```diff
@@ -0,0 +1,78 @@
+input-group-addon2 {
+    /*.input-group-addon,
+    .input-group-btn,
+    .input-group .form-control {*/
+        display: table-cell;
+    /*}*/
+
+    /*.input-group-addon,
+    .input-group-btn {*/
+        width: 1%;
+        white-space: nowrap;
+        vertical-align: middle;
+    /*}*/
+
+    /*.input-group-addon {*/
+        /*padding: 6px 12px;*/
+        font-size: 14px;
+        font-weight: normal;
+        line-height: 1;
+        color: #555;
+        text-align: center;
+        background-color: #eee;
+        border: 1px solid #ccc;
+        border-radius: 4px;
+    /*}*/
+}
+
+.input-group-addon2:not(:first-child):not(:last-child) {
+    /*.input-group-addon:not(:first-child):not(:last-child),
+    .input-group-btn:not(:first-child):not(:last-child),
+    .input-group .form-control:not(:first-child):not(:last-child) {*/
+        border-radius: 0;
+    /*}*/
+}
+
+.input-group-addon2:first-child {
+    /*.input-group .form-control:first-child,
+    .input-group-addon:first-child,
+    .input-group-btn:first-child > .btn,
+    .input-group-btn:first-child > .btn-group > .btn,
+    .input-group-btn:first-child > .dropdown-toggle,
+    .input-group-btn:last-child > .btn:not(:last-child):not(.dropdown-toggle),
+    .input-group-btn:last-child > .btn-group:not(:last-child) > .btn {*/
+        border-top-right-radius: 0;
+        border-bottom-right-radius: 0;
+    /*}*/
+}
+
+.input-group-addon2:first-child {
+    /*.input-group-addon:first-child {*/
+        border-right: 0;
+    /*}*/
+}
+
+.input-group-addon2:last-child {
+    /*.input-group .form-control:last-child,
+    .input-group-addon:last-child,
+    .input-group-btn:last-child > .btn,
+    .input-group-btn:last-child > .btn-group > .btn,
+    .input-group-btn:last-child > .dropdown-toggle,
+    .input-group-btn:first-child > .btn:not(:first-child),
+    .input-group-btn:first-child > .btn-group:not(:first-child) > .btn {*/
+        border-top-left-radius: 0;
+        border-bottom-left-radius: 0;
+    /*}*/
+}
+
+.input-group-addon2:last-child {
+    /*.input-group-addon:last-child {*/
+        border-left: 0;
+    /*}*/
+}
+
+
+/* CEVEAT1: Children under `input-group-addon2` classed tag, won't render like as if they children of `input-group-addon` */
+.input-group-addon2>*:not(.input-group-with-selectize) {
```

Comment:
Are we supposed to edit this component's css whenever we introduce a new component that might be used in `unit` field?